package-lock.json

```diff
@@ -0,0 +1,11077 @@
+{
+  "name": "betaflight-configurator",
```

Comment:
npm is not used in this project - please remove.

Also, please make sure you test everything with yarn before oipening pull requests - failing to do so will result in errors.